ui/docs/manual/deploying/queue-config.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Queue Configuration
+---
+
+# Queue Configuration
+
+This section describes configuration values specific to the Queue service.
+
+## Artifact Storage
+
+Until everything uses the Object Service, the queue service must be configured to support S3 artifacts.
+That involves the following Helm values:
+
+* `queue.artifact_region`
+* `queue.public_artifact_bucket`
+* `queue.private_artifact_bucket`
+* `queue.sign_public_artifact_urls` (optional, default false)
+* `queue.public_artifact_bucket_cdn` (optional)
+* `queue.aws_access_key_id`
+* `queue.aws_secret_access_key`
```

Comment:
Do we suggest defaults or typical values for Helm values?

Reply:
We don't really document Helm values in any consistent way right now.  None of the above would have defaults, aside from the one that is documented to do so.  And below `queue.task_claim_timeout`'s default is documented with some context on what appropriate values may be.

Or maybe I misunderstood the question?

Reply:
No, I think you understood. If I was setting these values, I'd be curious if ``queue.public_artifact_bucket`` was ``my-bucket-name`` or ``https://my-bucket-name.s3.us-east-2.amazonaws.com``. Just having the list is a huge improvement, and we can see if it makes sense to the users later.

Reply:
Good point -- I think at some point we will regularize all of this, at which time that might be a good improvement.